9.2/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM centos:centos7
+
+# PostgreSQL image for OpenShift.
+# Volumes:
+#  * /var/lib/psql/data   - Database cluster for PostgreSQL
+# Environment:
+#  * $POSTGRESQL_USER     - Database user name
+#  * $POSTGRESQL_PASSWORD - User's password
+#  * $POSTGRESQL_DATABASE - Name of the database to create
+#  * $POSTGRESQL_ADMIN_PASSWORD (Optional) - Password for the 'postgres'
+#                           PostgreSQL administrative account
+
+# Image metadata
+ENV POSTGRESQL_VERSION      9.2
+ENV IMAGE_DESCRIPTION       PostgreSQL 9.2
+ENV IMAGE_TAGS              postgresql,postgresql92
+ENV IMAGE_EXPOSE_SERVICES   5432:postgresql
+
+# These are relevant to PostgreSQL
+ENV PGDATA /var/lib/pgsql/data
+# TODO: We should consider if this should be changeable. Can affect performance.
+#       http://www.postgresql.org/docs/9.2/static/locale.html
+ENV LANG en_US.utf8
```

Comment:
Since we are not doing this for any other image, I would vote for nuking this line. You can set it in run-postgresql.sh (and users can change it by using -e)